mdakit_membcurv/tests/datafiles.py

```diff
@@ -0,0 +1,23 @@
+
+
+"""
+Location of datafiles for Membrane Curvature unit tests
+=======================================================
+
+MD simulations files stored in ``\\data`` sub-directory.
```

Comment:
I'd write it as
```reST
MD simulations files stored in ``data`` sub-directory.
```
and not start with a leading directory separator. (Also, I _think_ convention is to use Unix forward slashes as directory separators in documentation... but I might be biased.)